### DISCUSSION TITLE
Two step verification

### DISCUSSION THREAD

**Discussion by rudydepo** [geodesicsolutions-community/geocore-community#212] [Ideas]
Two-step verification will soon become a legal requirement for all logins in Europe.

**Comment by vicos59** [comment]
Hi Rudy,

The project is not being actively developed at this point in time. Pretty much just trying to keep it running as-is.

In addition to the codebase being quite old, this does not sound like a trivial upgrade. @blufyremedia is the only developer on the project. Maybe he can chime in if it is something he would take on.

**Reply by rudydepo** [reply]
If someone is interested, in developing we will fund it for a part and afterwards can be used for free; maybe as an addon .....

**Reply by vicos59** [reply]
@blufyremedia Are you interested, or can you recommend anyone for these folks to contact? @jonyo

**Reply by 1kiwi1** [reply]
> Hi Rudy,
> 
> The project is not being actively developed at this point in time. Pretty much just trying to keep it running as-is.
> 
> In addition to the codebase being quite old, this does not sound like a trivial upgrade. @blufyremedia is the only developer on the project. Maybe he can chime in if it is something he would take on.

Alternatively reach out to Lance at geomodules.com, he has a 2FA SMS Twilio Addon that will save you a lot of time and money in development.  Let us know how it goes.

**Reply by 1kiwi1** [reply]
> > Hi Rudy,
> > The project is not being actively developed at this point in time. Pretty much just trying to keep it running as-is.
> > In addition to the codebase being quite old, this does not sound like a trivial upgrade. @blufyremedia is the only developer on the project. Maybe he can chime in if it is something he would take on.
> 
> Alternatively reach out to Lance at geomodules.com, he has a 2FA SMS Twilio Addon that will save you a lot of time and money in development. Let us know how it goes.

Note: there's other ways of implementing MFA, all subject to SWOT.  This solution already exists, and got a mention at around $150.

**Comment by blufyremedia** [comment]
> Two-step verification will soon become a legal requirement for all logins in Europe.

Hello @rudydepo 

If that is still available, the SMS would be one option. However, as there are also many regulatory changes sweeping down the pipeline around the globe regarding SMS, so there may be hurdles (good hurdles!) to jump if you go that route. We've also done various SMS features for clients.

That said, more commonplace these days would be the use of an authenticator app, which can certainly be integrated. This can also alleviate additional transactional costs associated with SMS, along with addressing some other delivery issues and concerns. While not currently a feature of the core build, it can be pursued if you would like to connect to discuss further.
